src/ai/backend/manager/server.py

```diff
@@ -350,19 +359,37 @@ async def etcd_ctx(root_ctx: RootContext, etcd_config: EtcdConfigData) -> AsyncI
 
 @actxmgr
 async def unified_config_ctx(
-    root_ctx: RootContext, local_config: ManagerLocalConfig, local_cfg_loader: AbstractConfigLoader
+    root_ctx: RootContext, base_local_config: ManagerLocalConfig
 ) -> AsyncIterator[ManagerUnifiedConfig]:
-    etcd_loader = LegacyEtcdLoader(root_ctx.etcd)
-    raw_shared_cfg = await etcd_loader.load()
-    shared_cfg = ManagerSharedConfig(**raw_shared_cfg)
+    loaders: list[AbstractConfigLoader] = []
+
+    if base_local_config._config_path:
```

Comment:
Make the private value inaccessible from the outside.

Reply:
Private attributes such as `_config_path` are enforced to start with an underscore in Pydantic.